server/database/data/tasksFakeData.json

```diff
@@ -0,0 +1,30 @@
+[
+ { 
+  "taskName": "Installing a text editor",
+  "taskDescription": "How to Installing a text editor" 
+ },
```

Comment:
add it on your json as a simulate for fake data, also task description didn't express about paragraph as we said, it should be longer , you can copy paragraphs from technical task link that I posted yesterday .